proto/viam/app/v1/app.proto

```diff
@@ -122,10 +153,96 @@ message Organization {
   google.protobuf.Timestamp created_on = 3;
 }
 
+message OrganizationMember {
+  string user_id = 1;
+  google.protobuf.Timestamp date_added = 2;
+}
+
 message ListOrganizationsResponse {
   repeated Organization organizations = 1;
 }
 
+message OrganizationInvite {
+  string invite_id = 1;
+  string organization_id = 2;
+  string email = 3;
+  google.protobuf.Timestamp created_on = 4;
+}
+
+message CreateOrganizationRequest {
+  string user_id = 1;
```

Comment:
Remove `user_id`. We can handle adding the user making the API call to the new org.